deploy/sku-limits-configmap.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: sku-limits-managed-api-service
+data:
+  rate_limit: |-
+    {
+      "RHOAM SERVICE SKU": {
```

Comment:
@sergioifg94 this SKU configmap template would only be used for dev correct? If so we should probably look at reducing the reqPerUnit to something that can be easily tested.

It has yet to be decided what unit we'll be limiting on in production: `second` or `minute` but for dev minute makes most sense.

Reply:
Yes it would. For now I simply copied the example values from the ticket, but the operator won't reconcile back manual changes that are made to this ConfigMap (hive will, in the case of addon installations), so it's possible for us to tweak this values in development regardless of the initial value

Reply:
Also, this is my fault for specifying an incorrect name in the configmap example in the Jira but the parameter is `requests_per_unit` rather than `reqPerUnit`. See official docs: https://github.com/envoyproxy/ratelimit#rate-limit-definition

Reply:
> Yes it would. For now I simply copied the example values from the ticket, but the operator won't reconcile back manual changes that are made to this ConfigMap (hive will, in the case of addon installations), so it's possible for us to tweak this values in development regardless of the initial value

Sure, that makes sense. I don't feel particularly strongly about it but I think having spoke with @MStokluska on a related Jira we decided to go with 20 requests per minute for now

Reply:
> the parameter is `requests_per_unit` rather than `reqPerUnit`

That's handled anyways, as the operator will set the correct field in the ConfigMap that's mounted to the Rate Limit deployment:
https://github.com/integr8ly/integreatly-operator/blob/0cdf5dac9b9fcbcf6c0b5f0b442b11cccf7de348/pkg/products/marin3r/rateLimitService.go#L43
https://github.com/integr8ly/integreatly-operator/blob/0cdf5dac9b9fcbcf6c0b5f0b442b11cccf7de348/pkg/products/marin3r/rateLimitService.go#L99
Unless we want the fields to match in both ConfigMaps. In that case I could update that field

> we decided to go with 20 requests per minute for now

Sounds good, I'll update the development ConfigMap for this value

Reply:
> Unless we want the fields to match in both ConfigMaps. In that case I could update that field
> > we decided to go with 20 requests per minute for now

For consistency it might make sense to have them matching unless there is a valid reason not to

Reply:
Cool, I just updated this PR and [the managed-tenants MR](https://gitlab.cee.redhat.com/service/managed-tenants/-/blob/2e0bde81b7764d8bae4a697dc10d4c74245cfa93/addons/managed-api-service/metadata/stage/sku-limits.yaml.j2) to use the `requests_per_unit` field

Reply:
Thanks @sergioifg94 changes look good. In the process of validating now